src/utils/neynar/user/neynarGetVerifiedAddressesForFid.ts

```diff
@@ -0,0 +1,35 @@
+import { version } from '../../../version';
+import { FetchError } from '../exceptions/FetchError';
+import { NEYNAR_DEFAULT_API_KEY } from '../frame/neynarFrameFunctions';
+
+export async function neynarGetVerifiedAddressesForFid(
+  fid: number,
+  apiKey: string = NEYNAR_DEFAULT_API_KEY,
+): Promise<string[]> {
+  const options = {
```

Comment:
If we are starting making many Neynar call, it feels like we might want to abstract the creation of this optios.

Reply:
okay makes sense! I will abstract this part